lib/Factory/UserFactory.php

```diff
@@ -415,4 +415,46 @@ public function query($sortOrder = [], $filterBy = [])
 
         return $entries;
     }
+
+    /**
+     * Get a count of users, with respect to the logged-in user
+     * @return int
+     */
+    public function count(): int
+    {
+        $params = [];
+        $sql = '
+        SELECT COUNT(DISTINCT `user`.userId) AS countOf
+          FROM `user`
+            INNER JOIN `lkusergroup`
+            ON `lkusergroup`.userId = `user`.userId
+            INNER JOIN `folder`
+            ON `folder`.folderId = `user`.homeFolderId
+            INNER JOIN `group`
+            ON `group`.groupId = `lkusergroup`.groupId
+              AND `group`.isUserSpecific = 1
+        ';
+
+        // Super admins should get a count of all users in the system.
+        if (!$this->getUser()->isSuperAdmin()) {
+            // Non-super admins should only get a count of users in their group
+            $sql .= '
+                WHERE user.userId IN (
```

Comment:
`user.userId` should be \`user\`.userId instead (with backticks).

Reply:
Thanks, good catch.